lib/ci/ci_type_parser.js

```diff
@@ -86,7 +86,7 @@ const CI_TYPES = new Map([
   [LITE_PR_PIPELINE, {
     name: 'Lite PR Pipeline',
     jobName: 'node-test-pull-request-lite-pipeline',
-    pattern: /job\/node-test-pull-request-lite-pipeline\/(\d+)\/pipeline/,
+    pattern: /node-test-pull-request-lite-pipeline\/(\d+)\/pipeline/,
```

Comment:
This is an unrelated change? (If it allows the GitHub bot posted links then that's 💯 👍 but should be in its own commit and probably have tests if that's the intention.)